src/CockpitSDK.js

```diff
@@ -282,6 +283,16 @@ class CockpitSDK {
     };
   }
 
+  // @param {string} formName
+  // @param {Request} data
+  formSubmit(formName, data) {
```

Comment:
Since the property `form` is "the thing" of this method I'd rather move it to a separate argument.

Also, `options` instead of `data` to keep consistency w/ the rest of the methods.

```suggestion
  formSubmit(formName, form, options = {}) {
```

Reply:
I see your point, and it's something I considered while implementing it, but the data intended for cockpit it the form with the options, so it makes sense to me to keep it all as it is sent to cockpit.

Besides `collectionSave` also does the same thing, right? It has data and options and they are all kept in the same place.

Reply:
Actually no, 😄 . It's like I'm suggesting.
Since this is an abstraction over the Cockpit API we can add some flavor.
Notice that on `collectionSave` we have:
`body: this.stringifyOptions({ data })` not like you have here: `body: this.stringifyOptions(data)`.
Meaning that we're "converting" the argument to the right cockpit attribute in this case, `data`.
Here the attribute `form` is our `data`.

So the user of the CockpitSDK doesn't need to memorize that to the `formSubmit` it needs to send an object with a `form` attribute and to `collectionSave` an object with a `data` attribute.

Reply:
BTW is there any possible options other than `form`?